src/example-relay/src/pages/_app.js

```diff
@@ -102,9 +71,21 @@ const styles = sx.create({
   box: {
     backgroundColor: '#FFF8E1',
     maxWidth: '100%',
-    padding: 12,
+    padding: 'var(--space-small)',
   },
   boxWarning: {
     backgroundColor: '#FFCA28',
   },
+  padMedium: {
+    padding: 'var(--space-small)',
+    [tablet]: {
```

Comment:
This is an interesting technique. I kinda like it. :)

Reply:
Yeah, it seems like the best way to keep your breakpoints in sync throughout the app 🙂